src/main/java/frc/robot/subsystems/OldClimber.java

```diff
@@ -123,11 +96,13 @@ public boolean isArmFullyUp() {
         return false;
     }
 
+    //negative percent output values bring climber in, positive bring it out.
```

Comment:
This should be a Javadoc, ask a mentor for a mini-lesson if you are not sure how to write one